stable/hoard/README.md

```diff
@@ -17,12 +17,12 @@ $ helm install --name my-release stable/hoard
 The [configuration](#configuration) section below lists all possible parameters that can be configured during installation.
 
 
-### S3 Example
+### Cloud Example
 
-To deploy with an S3 backend, use the following command. Please first create appropriate [AWS Credentials](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html) and apply them to the Kubernetes secret `s3-credentials`.
+Fetch the appropriate credentials ([AWS](https://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html), [Azure](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-manage), [GCP](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) and apply them to the Kubernetes secret `cloud-credentials`. To deploy with a cloud backend, use the following command where `$CLOUD` is either `aws`, `azure` or `gcp`:
```

Comment:
since we're opinionated about the secret keyName we should add a bit more text outlining how exactly to do this. I'd have the topline example be just filesystem and then add individual sections for each supported cloud provider after the params table. So, e.g., for AWS, get creds, save them as a secret with this keyName and that keyName, then the helm install, etc. for GCP and Azure.